salsah2xml.py

```diff
@@ -175,7 +175,7 @@ def upper_camel_case(str) -> str:
     return camel_case(str, 'upper')
 
 
-def process_richtext(utf8str: str, textattr: str = None, resptrs: List = []) -> (str, str):
+def process_richtext(utf8str: str, projectname: str, textattr: str = None, resptrs: List = []) -> (str, str):
```

Comment:
The order of variables seems odd. Maybe:
```suggestion
def process_richtext(projectname: str, utf8str: str, textattr: str = None, resptrs: List = []) -> (str, str):
```

Saying this, would it make sense to move the entire function into Salsah class (as `process_value` and others are already). This way, `self.projectname` would be available without the need to pass another variable to the function?